teuthology/kill.py

```diff
@@ -84,6 +86,10 @@ def kill_job(run_name, job_id, archive_base=None, owner=None, skip_nuke=False):
                 "Please pass --owner <owner>.")
         owner = job_info['owner']
     kill_processes(run_name, [job_info.get('pid')])
+    if 'machine_type' in job_info:
```

Comment:
maybe ``if job_info.get('machine_type')`` ?

Reply:
I think if it is present in the dict but has a value of `None`, we may want to have that reported via prometheus; it would indicate something very odd went wrong.